pkg/operator/ceph/cluster/monitoring.go

```diff
@@ -87,7 +87,7 @@ func (c *ClusterController) configureCephMonitoring(cluster *cluster, clusterInf
 	// note: the error return below is ignored and is expected to be removed from the
 	//   bucket library's `NewProvisioner` function
 	bucketController, _ := bucket.NewBucketController(c.context.KubeConfig, bucketProvisioner)
-	go bucketController.Run(cluster.stopCh)
+	go bucketController.Run(cluster.stopCh) //nolint
```

Comment:
We should handle this error, at least to log if it fails. So we could define a very small goroutine wrapper.

Reply:
Done.